README.md

```diff
@@ -84,6 +84,7 @@ We recommend installing uv for dependency management when developing for `aiida-
 git clone https://github.com/stfc/aiida-mlip
 cd aiida-mlip
 uv sync  # Create a virtual environment and install dependencies
+uv sync --extra mace # Install mace seperately
```

Comment:
This doesn't need to be a separate command to the one above. I'd maybe also comment that MACE is used in the tests